cmd/root.go

```diff
@@ -150,8 +150,10 @@ func resolveContext(context string) (server, namespace string, err error) {
 	// use the default context where context is empty
 	if context == "" {
 		if rawConfig.CurrentContext == "" {
```

Comment:
Instead of the nested `if`, we may want to do something along the lines of 

```
if len(rawConfig.Clusters) == 0 {
  // User likely does not have a kubeconfig file.
  return "", "", fmt.Errorf("No current context found. Make sure a kubeconfig file is present")
}

if rawConfig.CurrentContext == "" {
  // return appropriate error for no active context set.
}
```

Otherwise, I believe the error on line 163 will be landed.

Reply:
@jessicayuen My understanding is that "no active context" is not considered an error by Kubernetes itself. From https://github.com/coreos/tectonic-installer/issues/1906, it says
> kubectl doesn't mind, but other clients may be sensitive

I tested the fix included in this PR, and it worked for my cluster, which does not have the active context configured. This also matches the behavior prior to d252456c2cf1d1bace58b96162ad6180b8a0bd9b

In this case, `context` is set to the empty string `""`, and the matching context (`rawConfig.Contexts[""]`) is set on https://github.com/ksonnet/ksonnet/pull/253/files#diff-ff7686b39bf90dc2520886fb874371a4L159

I'll change the nested if to an `&&` condition to make the error case more obvious.